Demo/VIper-v2-Demo/VIper-v2-Demo/Application/Initializers/ThemeInitializer.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ThemeInitializer.swift
+//  Pokedex
+//
+//  Created by Filip Beć on 27/04/2017.
+//  Copyright © 2017 Filip Beć. All rights reserved.
+//
+
+import UIKit
+
+class ThemeInitializer: NSObject, Initializable {
+
+    @objc func performInitialization() {
+        UINavigationBar.appearance().tintColor = UIColor.white
```

Comment:
Setting appearance directly to UINavigationBar is not so good approach. It would be better to use:

```swift
UINavigationBar.appearance(whenContainedInInstancesOf: [PokedexNavigationController.self]).barTintColor = .white
```

Reasoning behind this is that some native components like iCloud/Photos picker uses default UINavigationController and they conform to appearance protocol partially (it will change text color, but they wont use translucency) and then it looks awful. So with above stated approach, you would have empty subclass of UINavigationController `PokedexNavigationController` which you would use inside your app (not UINavigationController) and in that way you wont mess with native components rendering.